### ISSUE TITLE
Reporting 'unsupported file types' as asked for by command

### ISSUE THREAD

**Issue by Jeehut** [SDGGiesbrecht/Workspace#599]
### Question

I just ran `workspace refresh` to see what would happen in [HandySwift](https://github.com/Flinesoft/HandySwift)'s root directly and it reported the following 'unsupported file types':

```shell
podspec (HandySwift.podspec)
stencil (.sourcery/LinuxMain.stencil)
xcplayground (UsageExamples.playground/contents.xcplayground)
```

The output further states that 'If these are standard file types, please report them' so I'm reporting them. Though, I'm not sure what 'standard file types' are, but interpreting it as 'common types for a Swift package' I think it applies to all of these.

### Documentation Suggestion

Maybe be more clear about what a `standard file type` is? And I also don't understand why I should report it in the first place, or should I just ignore them via `repository.ignoredFileTypes` as suggested as an alternative? It's also unclear, by which issue type these should be reported, as bugs?

**Comment by SDGGiesbrecht** [comment]
Workspace doesn’t recognized the file type and cannot safely process it (such as not knowing the comment format and being unable to apply a file header), so this is notifying you that such files have been skipped.

Reporting it allows us to teach Workspace what to do with such a file, (even if that ends up just being to ignored them).

**Comment by Jeehut** [comment]
I guess then we need to add these file types to be ignored by Workspace, I'm assuming, given that we should probably not make any header or formatting checks on them (except maybe later for the `.podspec` type as it's plain Ruby?).

**Comment by SDGGiesbrecht** [comment]
#608 adds these file types.

**Comment by SDGGiesbrecht** [comment]
Released in 0.35.3.
